app/static/js/request/new-request-agency.js

```diff
@@ -142,18 +142,16 @@ $(document).ready(function () {
         if ($('#email').parsley().isValid() ||
             $('#phone').parsley().isValid() ||
             $('#fax').parsley().isValid() ||
-            ($('#address-line-1').parsley().isValid() &&
-            $('#state').parsley().isValid() &&
-            $('#zipcode').parsley().isValid() &&
-            $('#city').parsley().isValid())
+            ($('#address-line-1').parsley().isValid() && $('#state').parsley().isValid() && $('#zipcode').parsley().isValid() && $('#city').parsley().isValid())
             &&
-            ($('#request-agency').parsley().isValid() &&
-            $('#request-title').parsley().isValid() &&
+            ($('#request-title').parsley().isValid() &&
```

Comment:
You removed `request-agency` because there is no agency to choose from right? Does this mean, it can also be removed from [here](https://github.com/CityOfNewYork/NYCOpenRecords/pull/136/files#diff-88d5d9727dd0b0cec91064b9129d63daR64)?